.github/workflows/dhis2-verify-app.yml

```diff
@@ -127,13 +127,17 @@ jobs:
               uses: cypress-io/github-action@v2
               with:
                   start: yarn cypress:start
-                  command: yarn d2-utils-cypress run --stub --serverMinorVersion 37 --appStart ''
                   wait-on: 'http://localhost:3000'
                   wait-on-timeout: 300
               env:
-                  cypress_dhis2_base_url: https://debug.dhis2.org/dev
-                  cypress_record_key: ${{ secrets.CYPRESS_RECORD_KEY }}
                   GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+                  CYPRESS_RECORD_KEY: ${{ secrets.CYPRESS_RECORD_KEY }}
+                  # Used by the app platform
+                  DHIS2_BASE_URL: https://debug.dhis2.org/dev
```

Comment:
I thought that because we call [`setDhis2BaseUrlToLocalStorage`](https://github.com/dhis2/cli-utils-cypress/blob/alpha/packages/cypress-commands/src/helper/dhis2BaseUrl.js#L15) and use the `dhis2BaseUrl` Cypress env variable when we [enable the `autoLogin` plugin](https://github.com/dhis2/cli-utils-cypress/blob/alpha/packages/cypress-commands/src/setups/enableAutoLogin.js#L17-L20) and when we [enable the `networkShim` plugin](https://github.com/dhis2/cli-utils-cypress/blob/alpha/packages/cypress-commands/src/setups/enableNetworkShim/index.js#L33-L35) we don't need to duplicate the `DHIS2_BASE_URL` here?

Reply:
Or is it for the `yarn start` command that the Cypress action runs?

Reply:
Trying without it in a76a44b and will revert it if it doesn't work.

Reply:
It works without this env variable, so I think we can leave it out.